tests/unit-tests/admin/test-class-sensei-onboarding.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * This file contains the Sensei_Onboarding_Test class.
+ *
+ * @package sensei
+ */
+
+/**
+ * Tests for Sensei_Onboarding_Test class.
+ *
+ * @group onboarding
+ */
+class Sensei_Onboarding_Test extends WP_UnitTestCase {
+	/**
+	 * Set up before each test.
+	 */
+	public function setup() {
+		parent::setup();
+
+		$this->onboarding_instance = new Sensei_Onboarding();
+
+		// Save original current screen.
+		global $current_screen;
+		$this->original_screen = $current_screen;
+	}
+
+	/**
+	 * Tear down after each test.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+
+		// Restore current screen.
+		global $current_screen;
+		$current_screen = $this->original_screen;
+	}
+
+	/**
+	 * Testing the onboarding class to make sure it is loaded.
+	 */
+	public function testClassInstance() {
+		$this->assertTrue( class_exists( 'Sensei_Onboarding' ), 'Sensei Admin class does not exist' );
```

Comment:
'Admin' -> 'Onboarding' update in the message.

Reply:
Oops! https://github.com/Automattic/sensei/pull/3093/commits/9841b5341d5d2b4c8a438add4ff1c9d4aa859b05